src/include/ipc/info.h

```diff
@@ -33,6 +33,7 @@
 #define SOF_IPC_INFO_LOCKS		BIT(1)
 #define SOF_IPC_INFO_LOCKSV		BIT(2)
 #define SOF_IPC_INFO_GDB		BIT(3)
+#define SOF_IPC_INFO_IMR_RESTORE	BIT(4)
```

Comment:
This sounds awfully Intel-specific. Why make any mentions of IMR here?

Should this be SOF_IPC_INFO_PLATFORM_SUSPEND ?